changelogs/fragments/60530-facts-info-rename.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- The ``digital_ocean_sshkey_facts`` module has been renamed to ``digital_ocean_sshkey_info``.
```

Comment:
If you do changes to the return values, you also need to document them somewhere. A note about that should be here and in the porting guide.

(If you're deprecating / adding new module, that won't be needed.)

Reply:
Will update this if someone asks to change the return value

Reply:
The problem is that you cannot use `ansible_facts` for the `_info` module. Everything returned in `ansible_facts` will be made available globally.

So maybe it is better to deprecate the `_facts` module and replace it with a new `_info` module which uses `data` (as the other modules)?